app/models/deleted_observation.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 class DeletedObservation < ApplicationRecord
   belongs_to :user
```

Comment:
Considered validating presence `on: :create`, however not sure it's strictly needed.